src/interfaces/tooltipNew.d.ts

```diff
@@ -76,6 +76,7 @@ export interface IUserGuideTooltipProps
   theme?: TooltipTheme;
   placement?: PopperJS.Placement;
   containerName?: string;
+  disableSpotlightClicks?: boolean;
```

Comment:
`disabledSpotlightPointerEvents` maybe
hmm, still it will be possible to do some action with the keyboard. Is this solution good enough for us?

Reply:
I changed the prop name.

Yes, I think disabling the mouse events is good enough.